Cataclysm/Druid.lua

```diff
@@ -648,8 +682,8 @@ end)
 
 
 spec:RegisterStateExpr("calc_rake_dpe", function()
-    local rake_dpe = action.rake.damage + action.rake.tick_damage*(math.floor(min(aura.rake.duration,ttd)/aura.rake.tick_time))/action.rake.cost
-    return rake_dpe
+    local rake_dpe = action.rake.damage + action.rake.tick_damage*(math.floor(min(aura.rake.duration,ttd)/aura.rake.tick_time))
```

Comment:
dpe = damage per energy

If you dont divide by energy then its just damage. Doesnt make sense to split this up while keeping the name

Reply:
You're right, the name is confusing in this way, the objective of the change is to correctly divide the value, the division is done before the addition so, the division only applied to tick damage and not to the initial damage, i.e. initial damage + tick DPE, this inflates the DPE a lot.
I can change the name np